src/synthesize.jl

```diff
@@ -216,13 +216,15 @@ function format_A_X(metallicity::Real=0.0, abundances::Dict=Dict();
                 throw(ArgumentError("$el isn't a valid atomic symbol."))
             elseif Korg.atomic_numbers[el] in keys(abundances)
                 throw(ArgumentError("The abundances of $el was specified by both atomic number and atomic symbol."))
+            elseif el != "H"
+                clean_abundances[Korg.atomic_numbers[el]] =  abund
             end
-            clean_abundances[Korg.atomic_numbers[el]] =  abund
```

Comment:
Is this change necessary?  (I think that it's not, at least if you make the change recommended above.)